src/query/service/src/sql/planner/plans/prewhere.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::Scalar;
+use crate::sql::optimizer::ColumnSet;
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct Prewhere {
```

Comment:
`Prewhere` isn't a real plan, I think it's confusing to put it in the plans dir.

Reply:
How about put it in planner/logical_get.rs.

Reply:
OR in src/query/service/src/sql/common/mod.rs

Reply:
> How about put it in planner/logical_get.rs.

It's ok